StaffManagement.java

```diff
@@ -3,24 +3,65 @@
 
 public class StaffManagement
 {
-    private List<Staff> staffs;
+    List<Staff> staffs;

```

Comment:
Is there any reason why to change the list of staffs from private to package-private?

Reply:
No difference that I can see between either version.